kpatch-build/kpatch-build

```diff
@@ -165,7 +165,7 @@ find_parent_obj() {
 }
 
 find_kobj() {
-	arg=$1
+	arg=$(echo $1 |sed 's,^\S*\.\./,,g')
```

Comment:
Oh, it's my fault. I checked changed_objs and found 'arch/x86/kvm/../../../virt/kvm/kvm_main.o'.
Please ingore this patch.
Sorry.
Need I re-open a pull-request?